far/FarRus.hlf.m4

```diff
@@ -2879,96 +2879,108 @@ $ #Формат командной строки#
     #Alt+F8#             ~Изменить текущую позицию~@ViewerGotoPos@
-    #Alt+F9#             Изменение размеров окна Far Manager
+    #Alt+F9#             Изменить размеры окна Far Manager
     #Alt+Shift+F9#       Вызов диалога ~настройки программы просмотра~@ViewerSettings@
     #NumPad5,F3,F10,Esc# Выход
-    #Ctrl+F10#           Позиционировать на текущий файл без выхода.
-    #F11#                Вызвать меню "~Команды плагинов~@Plugins@"
-    #Alt+F11#            Показать ~историю просмотра~@HistoryViews@
-    #Gray +#             Перейти к следующему файлу
-    #Gray -#             Перейти к предыдущему файлу
+    #Ctrl+F10#           Перейти на текущий файл на активной файловой панели.
+    #F11#                Показать меню ~Команд плагинов~@Plugins@
+    #Alt+F11#            Показать ~историю~@HistoryViews@ просмотра и редактирования
+    #Gray +#             Просматривать следующий файл на активной файловой панели
+    #Gray -#             Просматривать предыдущий файл на активной файловой панели
     #Ctrl+O#             Показать пользовательский экран
-    #Ctrl+Alt+Shift#     Временно показать пользовательский экран
-                       (работает до тех пор, пока удерживаем эти
-                       клавиши)
-    #Ctrl+B#             Спрятать/Показать линейку функциональных
-                       клавиш
-    #Ctrl+Shift+B#       Спрятать/Показать статусную строку
-    #Ctrl+S#             Спрятать/Показать полосу прокрутки
-    #Alt+BS, Ctrl+Z#     Возврат к предыдущей позиции
+    #Ctrl+Alt+Shift#     Временно показать пользовательский экран (пока
+эти клавиши нажаты)
+    #Ctrl+B#             Спрятать/показать полосу функциональных клавиш внизу экрана
+    #Ctrl+Shift+B#       Спрятать/показать статусную строку
+    #Ctrl+S#             Спрятать/показать полосу прокрутки
+    #Alt+BS, Ctrl+Z#     Вернуться к предыдущей позиции
     #ПравыйCtrl+0..9#    Установить закладку 0..9 в текущей позиции
     #Ctrl+Shift+0..9#    Установить закладку 0..9 в текущей позиции
     #ЛевыйCtrl+0..9#     Перейти к позиции закладки 0..9
-
     #Ctrl+Ins, Ctrl+C#   Скопировать выделенный текст в буфер обмена.
-    #Ctrl+U#             Сбросить выделение текста.
-    #Shift+Клик мыши#    Ручное выделение текста.
-                       Если курсор мыши находится над текстом, первый
-                       клик задаёт начало области выделения текста,
-                       второй клик - конец области. После первого клика
-                       можно использовать любые клавиши навигации, чтобы
-                       сделать видимым конец области. Второй клик может
-                       быть раньше первого по тексту.
+Текст может быть выделен вручную или в результате ~поиска~@ViewerSearch@.
+    #Ctrl+U#             Сбросить выделение текста
+    #Shift+Клик мыши#    Выделить текст вручную
+                       Первый клик мыши задаёт начало выделенной
+области, второй клик -– конец области. После первого клика можно
```

Comment:
Are both `-` and `–` needed?

Reply:
Thank you for noticing. You are very polite. :) It's an error, result of MS Word autocorrect. I changed it to plain `--` to imitate typographic M-dash in a monospaced font.